src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const miniDiscount = 20;
+  const basicDiscount = 50;
+
+  let totalAmount = 0;
+
+  if (days < 3) {
+    totalAmount = days * dayPrice;
```

Comment:
Don't repeat yourself: you've repeated this calculation `days * dayPrice` 3 times, let's use it only once